com.unity.render-pipelines.high-definition/Editor/Material/Unlit/BaseUnlitGUI.cs

```diff
@@ -249,12 +247,15 @@ public static void SetupBaseUnlitKeywords(this Material material)
 
             // DoubleSidedGI has to be synced with our double sided toggle
             var serializedObject = new SerializedObject(material);
+            bool doubleSidedGI = false;
             if (doubleSidedGIMode == DoubleSidedGIMode.Auto)
-                material.doubleSidedGI = doubleSidedEnable;
+                doubleSidedGI = doubleSidedEnable;
             else if (doubleSidedGIMode == DoubleSidedGIMode.On)
-                material.doubleSidedGI = true;
+                doubleSidedGI = true;
             else if (doubleSidedGIMode == DoubleSidedGIMode.Off)
-                material.doubleSidedGI = false;
+                doubleSidedGI = false;
+            if (doubleSidedGI != material.doubleSidedGI)
+                material.doubleSidedGI = doubleSidedGI;
```

Comment:
a c++ PR to fix SetDoubleSidedGI would be a good idea I suppose

Reply:
For now it is fine, just add a comment here to explain that calling material.doubleSidedGI always dirty the amterial